pose_lookup/plugin.cpp

```diff
@@ -115,21 +159,27 @@ class pose_lookup_impl : public pose_prediction {
 
 private:
 	const std::shared_ptr<switchboard> sb;
-
 	mutable Eigen::Quaternionf offset {Eigen::Quaternionf::Identity()};
 	mutable std::shared_mutex offset_mutex;
 
-	/*pyh: reusing data_loading from ground_truth_slam*/
 	const std::map<ullong, sensor_types> _m_sensor_data;
 	ullong dataset_first_time;
 	time_type _m_start_of_time;
 	std::unique_ptr<reader_latest<time_type>> _m_vsync_estimate;
+
+	bool enable_alignment;
+	Eigen::Vector3f init_pos_offset {0};
+	Eigen::Matrix3f align_rot;
+	Eigen::Vector3f align_trans;
+	Eigen::Vector4f align_quat;
+	double align_scale;
+	std::string path_to_alignment = "./metrics/alignMatrix.txt";
```

Comment:
Similar to the `offload_data` plugin, I hard-coded the path to alignment. 

I think uses should be agnostic to these kinds of intermediate data. 

In the future, we could either provide the pre-computed alignment files for each data sequence, or set the output path of alignment computation that is pointing here.